src/types/featureFlags.ts

```diff
@@ -1,10 +1,17 @@
 // Use for type safety with GrowthBook
 // Add BE feature flags here to mirror that on GrowthBook
 // Note: key should mirror GrowthBook exactly as it is
+
+export interface cmConfigType {
```

Comment:
nit: the interface name should start with caps, and might be better to spell Cloudmersive out in full because it might not be clear what it stands for when we look at this in the future.